doc: matter: Fix custom cluster guide (#22262)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v3-0-1 v3-0-0

> KRKNWK-20308: The ``MyCluster.xml`` file example in the :ref:`ug_matter_creating_custom_cluster` user guide does not contain the ``ExtendedCommandResponse`` command.

Reviewer: ```suggestion
KRKNWK-20308: The ``MyCluster.xml`` file example in the :ref:`ug_matter_creating_custom_cluster` user guide does not contain the ``ExtendedCommandResponse`` command
```

>   Without this command, the file cannot be used properly in the rest of the guide.

Reviewer: ```suggestion
  You cannot use the file properly without this command.
```
